rita_common/src/debt_keeper/mod.rs

```diff
@@ -361,10 +414,15 @@ impl DebtKeeper {
     }
 
     fn save(&mut self) -> Result<(), IOError> {
+        let mut file_path: String = settings::get_rita_common().payment.debts_file;
         // convert to the serializeable format and dump to the disk
-        let serialized = serde_json::to_string(&debt_data_to_ser(self.debt_data.clone()))?;
-        let mut file = File::create(settings::get_rita_common().payment.debts_file)?;
-        file.write_all(serialized.as_bytes())
+        if file_path.ends_with("json") {
+            file_path.drain(0..file_path.len() - 4);
```

Comment:
that's an interesting way to do this. 👍🏻  You should put the new file path string into the settings.